values-cs/strings.xml

```diff
@@ -2,126 +2,126 @@
-    <string name="tap_to_play_random_move">Tap the grid to play a random move!</string>
+    <string name="how_to_play_2048_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_2048_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 2048?</string>
+    <string name="how_to_play_243_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 3 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_243_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 243?</string>
+    <string name="how_to_play_16384_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 nebo 4 dlaždice se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_16384_paragraph2">Po každém pohybu se přídají 2 nové dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 16384?</string>
+    <string name="how_to_play_blitz_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se sloučí do jedné pokuď mají stejné čísla.>/string>
+    <string name="how_to_play_blitz_paragraph2">Nové dlaždice se přidají po určitém intervalu, nikoliv po pohybu. Hra končí pokuď již nelze provést tah. Dokážeš udělat dlaždici s číslem 2048?</string>
+    <string name="how_to_play_fibonacci_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. 2 dlaždice se slučí do jedné, pokuď odpovídají pořadí.</string>
+    <string name="how_to_play_fibonacci_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelze provést tah.</string>
+    <string name="how_to_play_fibonacci_hint">Nápověda: 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89, 144&#8230;</string>
+    <string name="how_to_play_144_paragraph1">Přejeď doleva, doprava, nahoru nebo dolů k posunutí dlaždice po ploše. Jakýkoliv počet dlaždic se sloučí do jedné pokuď mají stejné čísla.</string>
+    <string name="how_to_play_144_paragraph2">Po každém pohybu se přidá nová dlaždice kdekoliv na ploše. Hra končí pokuď již nelye provést tah. Dokážeš udělat dlaždici s číslem 144?</string>
+    <string name="how_to_play_144_hint">Nápověda: 2 &#215; 2 &#215; 2 &#215; 2 &#215; 3 &#215; 3 = 144</string>
+    <string name="tap_to_play_random_move">Kliknutím zahraješ náhodný tah!</string>
 
     <!-- SHARING STRINGS -->
     <!-- The title of the activity chooser when you click the "Share" preference in the Settings -->
-    <string name="share_with">Share with</string>
+    <string name="share_with">sdílet</string>
```

Comment:
I think this is the last typo and then everything else looks great! (The string should be capitalized, I think). :)